src/test/FuzzerImpl.cpp

```diff
@@ -983,7 +983,11 @@ void
 TransactionFuzzer::initialize()
 {
     resetRandomSeed(1);
-    mApp = createTestApplication(mClock, getFuzzConfig(0));
+    auto cfg = getFuzzConfig(0);
+#ifdef BEST_OFFER_DEBUGGING
```

Comment:
you don't need this: as you're using a test configuration, it's already set

Reply:
Good point.  Removed.